training_mod_consts/src/options.rs

```diff
@@ -757,6 +760,9 @@ impl BuffOption {
                 BuffOption::MONAD_BUSTER => *FIGHTER_SHULK_MONAD_TYPE_BUSTER,
                 BuffOption::MONAD_SMASH => *FIGHTER_SHULK_MONAD_TYPE_SMASH,
                 BuffOption::POWER_DRAGON => 1,
+                BuffOption::WAFT_MINI => 1000,
```

Comment:
Could you use the associated lua consts rather than their values?

Reply:
~~I dont think there are associated lua consts, so i kinda trial-errored the best values.~~

I wasn't able to find the associated consts, but i just found params that give the thresholds in seconds
 
```
WorkModule::get_param_float(module_accessor, hash40("param_special_lw"), hash40("gass_max_time")) as i32; // value is 110
WorkModule::get_param_float(module_accessor, hash40("param_special_lw"), hash40("gass_large_time")) as i32; // value is 55
WorkModule::get_param_float(module_accessor, hash40("param_special_lw"), hash40("gass_middle_time")) as i32; // value is 15
```

this gets the value we want in seconds, but the internal waft counter counts the frames, so i guess ill just multiply by 60 and hopefully that should work

Reply:
https://github.com/jugeeya/UltimateTrainingModpack/pull/626/commits/7230fee5010305ad6c90ae4b02fa185749db14db